README.md

```diff
@@ -246,25 +249,37 @@ Which assets are generated depends on the target platform.
 - `.msi`: Global Windows installer (requires Administrator)
 - `-win32-${arch}.zip`: Convenience for techops team's that can't use `.msi` or use a package manager such as Chocolatey.
 
+### Linux
+
+- `.rpm`: Installer for Redhat
+- `.deb`: Installer for Ubuntu and Debian
+- `.tar.gz`: Convenience for easy automation on unix
+- `-linux-${arch}.zip`: Convenience for easy automation on win32-x64
+
 ## Auto Update
 
 - `hadron-auto-update-manager`: https://github.com/hadron-auto-update-manager
 
+### macOS
```

Comment:
This branch has grown large, would it be reasonable to extract the macOS / Windows / electron-prebuilt changes and get those merged independently?